astroid/node_classes.py

```diff
@@ -1051,6 +1051,19 @@ def postinit(self, targets=None, value=None):
         self.value = value
 
 
+class AnnAssign(mixins.AssignTypeMixin, Statement):
+    """class representing an AnnAssign node"""
+    _astroid_fields = ('target', 'value', 'annotation',)
```

Comment:
we should add `simple` and change orderubg to be in conformance with built-in `ast`.

```
>>> ast.parse('a: int').body[0]._fields
('target', 'annotation', 'value', 'simple')
```